content/refguide/saphana.md

```diff
@@ -20,3 +20,15 @@ For example, you have two associated entities, *Person* and *Address*, and they
 SAP HANA is case sensitive when doing string comparisons and checks. This is important when using functions such as `contains()`, `starts-with()`, and `ends-with()`, or when using equality(`=`) or inequality(`!=`) in xpath constraints.
 
 For example, `contains('OneTwo', 'one')` will return `false`.
+
+### Unlimited String Behavior
+
+SAP HANA does not support unlimited string when using equality(`=`) or inequality(`!=`) in xpath constraints. However, it supports on functions including `contains()`, `starts-with()`, and `ends-with()`.
+
+### Sorting on Very Long Strings
```

Comment:
Perhaps we can make `sorting` and `select distinct` a sub-header of Unlimited string behavior. We can then clarify the Long string behavior in the main section.

Reply:
changed it also added more